tools/release.rb

```diff
@@ -243,46 +199,49 @@ def generate_changelog
       changelog += value
     end
 
-    File.write(@package_changes_path, changelog)
+    File.write(RPM_CHANGES_FILE, changelog)
   end
 
   def finalize_news_file
-    @changelog_files.each do |file|
+    CHANGELOG_FILES.each do |file|
       content = File.read(file)
       # All changes for the next release are directly added below the headline
       # by the developers without adding a version line.
       # Since the version line is automatically added during release by this
       # method we can check for new bullet points since the last release.
       if content.scan(/# Machinery .*$\n+## Version /).empty?
-        content = content.sub(
-          /\n+/, "\n\n\n## Version #{@options[:version]} - #{@release_time} - #{@mail}\n\n"
-        )
+        content = content.sub(/\n+/, "\n\n\n## Version #{@release_version} - #{@release_time} - #{@mail}\n\n")
         File.write(file, content)
       end
     end
   end
 
   def add_default_rpm_changes_entry
-    file = "RPM_CHANGES"
+    file = File.join(Machinery::ROOT, "RPM_CHANGES")
     content = File.read(file)
 
-    regex = /(Machinery .{0,7}RPM Changelog).*?(\*|\n#)/m
-    content.sub!(regex, "\\1\n\n* update to version #{@options[:version]}\n\\2")
+    regex = /(Machinery RPM Changelog).*?(\*|\n#)/m
+    content.sub!(regex, "\\1\n\n* update to version #{@release_version}\n\\2")
 
     File.write(file, content)
   end
 
   def commit
-    # Build gem and send everything to IBS
-    Rake::Task["osc:commit"].invoke
-
-    # Set and commit git tag
-    Cheetah.run "git", "commit", "-a", "-m", "package #{@options[:version]}"
-    Cheetah.run "git", "tag", "-a", @tag, "-m", "Tag version #{@options[:version]}"
+    Cheetah.run "git", "commit", "-a", "-m", "package #{@release_version}"
+    Cheetah.run "git", "tag", "-a", @tag, "-m", "Tag version #{@release_version}"
     Cheetah.run "git", "push"
     Cheetah.run "git", "push", "--tags"
   end
 
+  def generate_development_version
+    # The development version RPMs have the following version number scheme:
+    # <base version>.<timestamp><os>git<short git hash>
+    timestamp = Time.now.strftime("%Y%m%dT%H%M%SZ")
+    commit_id = Cheetah.run("git", "rev-parse", "--short", "HEAD", :stdout => :capture).chomp
```

Comment:
It's a valid complaint, it should be `stdout: :capture` instead of `:stdout => :capture`.